torchbenchmark/models/maml/__init__.py

```diff
@@ -0,0 +1,85 @@
+from argparse import Namespace
+import math
+import time
+from tqdm import tqdm
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from .meta import Meta
+
+
+import random
+import numpy as np
+from pathlib import Path
+
+torch.manual_seed(1337)
+random.seed(1337)
+np.random.seed(1337)
+torch.backends.cudnn.deterministic = True
+torch.backends.cudnn.benchmark = False
+
+class Model:
+    def __init__(self, device=None, jit=False):
```

Comment:
i forget if we had a reason for making the default device 'cpu'.  i think it was at least important to not make it gpu for some reason. did you see other models use None?